core/dbt/events/types.py

```diff
@@ -1,6 +1,7 @@
 from abc import ABCMeta, abstractmethod
 from dataclasses import dataclass
-from typing import Any, List
+from typing import Any, List, Dict
+from dbt.ui import warning_tag
```

Comment:
We'll likely want this stuff to live in the events module eventually, but for now this is fine.